client/blocks/image-editor/style.scss

```diff
@@ -4,6 +4,13 @@
 	height: 100%;
 }
 
+.image-editor .notice {
+	// resorted to !important
+	// because there is
+	// no stronger declaration to overwrite .editor-media-modal .notice
+	margin-top: 0 !important;
```

Comment:
I think it'd be fine to add an extra class to beat the current specificity. Let's please always try to not include `!important` unless we're styling an email template.